docs/en/ingest-management/fleet/fleet-settings-output-kafka.asciidoc

```diff
@@ -5,6 +5,29 @@
 
 Specify these settings to send data over a secure connection to Kafka. In the {fleet} <<output-settings,Output settings>>, make sure that the Kafka output type is selected. 
 
+== Kafka output and using Logstash to index data to Elasticsearch
+
+If you are considering using {ls} to ship the data from `kafka` to {es}, please
+be aware Elastic is not currently testing this kind of setup.
+
+The structure of the documents sent from {agent} to `kafka` must not be modified by {ls}.
+We suggest disabling `ecs_compatibility` on both the `kafka` input and the `json` codec.
+
+Refer to {ls} output for {agent} for more details.
```

Comment:
```suggestion
Refer to the <<ls-output-settings,{ls} output for {agent}>> documentation for more details.
```
This will add a link to the Fleet [Logstash output page](https://www.elastic.co/guide/en/fleet/current/ls-output-settings.html).